src/live_restore/live_restore_fs.c

```diff
@@ -769,7 +769,8 @@ __live_restore_fh_find_holes_in_dest_file(
 
     data_end_offset = 0;
     WT_SYSCALL(((fd = open(filename, O_RDONLY)) == -1 ? -1 : 0), ret);
-    WT_ERR(ret);
+    if (ret != 0)
```

Comment:
This should be WT_RET(ret);

Reply:
Make that WT_RET_MSG(session, ret, "Failed to open file descriptor on %s", filename);

Reply:
https://github.com/wiredtiger/wiredtiger/pull/11331/commits/2407de456246f88bd3871751eef396e3b5539f8a

Reply:
Please correct me if I'm wrong, but I don't see `WT_RET_MSG` checking this return value in the same way `WT_RET` does. How can we remove this `if (ret != 0)`?

Reply:
Oh what, I had assumed they do WT_RET+MSG. My b.